growth-plans.md

```diff
@@ -0,0 +1,28 @@
+# Growth Plans
+
+The [CCC Project Progression Policy](project-progression-policy.md) explains
+that each [Incubation Stage](project-progression-policy.md#incubation-stage)
+project accepted by the CCC must have a growth plan to track its progress
+towards the [Graduation Stage](project-progression-policy.md#graduation-stage).
+
+A "growth plan" is a written set of [growth goals](#growth-goals) and a
+written set of actions intended to make forward progress towards those goals.
+The growth plan does not need to have a fixed timeline.
+
+The growth plan is reviewed when a project enters the Incubation or
+Graduation Stake, and also as part of the project's
+[Annual Review Process](project-progression-policy.md#iv-annual-review-process).
+
+## Growth Goals
+
+A project's growth goals are, at minimum, the CCC criteria for achieving 
+Graduation Stage.
+
+As a bit of history, the notion of project "incubation" and "graduation" were
+[introduced](https://docs.openstack.org/project-team-guide/introduction.html)
+by OpenStack and have since been followed by
+[CNCF](https://github.com/cncf/toc/blob/master/process/graduation_criteria.adoc)
+and other organizations, but the exact criteria vary by organization.
+
+Objective criteria for graduation are preferred rather than subjective criteria.
```

Comment:
```suggestion
Objective criteria for graduation are preferred rather than subjective criteria (TODO: update the CCC Graduation Stage criteria to be objective).
```